File: Seif/src/SlimBundle/Entity/Livraison.php
<?php

namespace SlimBundle\Entity;

use SBC\NotificationsBundle\Model\BaseNotification;

/**
 * Livraison
 */
class Livraison 
{
    /**
     * @var float
     */
    private $montant;

    /**
     * @var integer
     */
    private $etat;

    /**
     * @var \DateTime
     */
    private $date;

    /**
     * @var string
     */
    private $adresse;

    /**
     * @var float
     */
    private $longitude;

    /**
     * @var float
     */
    private $latitude;

    /**
     * @var string
     */
    private $nomclient;

    /**
     * @var string
     */
    private $nomlivreur;

    /**
     * @var integer
     */
    private $idLivraison;

    /**
     * @var \AppBundle\Entity\Utilisateur
     */
    private $idUser;

    /**
     * @var \AppBundle\Entity\Commande
     */
    private $idcommande;


    /**
     * Set montant
     *
     * @param float $montant
     *
     * @return Livraison
     */
    public function setMontant($montant)
    {
        $this->montant = $montant;

        return $this;
    }

    /**
     * Get montant
     *
     * @return float
     */
    public function getMontant()
    {
        return $this->montant;
    }

    /**
     * Set etat
     *
     * @param integer $etat
     *
     * @return Livraison
     */
    public function setEtat($etat)
    {
        $this->etat = $etat;

        return $this;
    }

    /**
     * Get etat
     *
     * @return integer
     */
    public function getEtat()
    {
        return $this->etat;
    }

    /**
     * Set date
     *
     * @param \DateTime $date
     *
     * @return Livraison
     */
    public function setDate($date)
    {
        $this->date = $date;

        return $this;
    }

    /**
     * Get date
     *
     * @return \DateTime
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * Set adresse
     *
     * @param string $adresse
     *
     * @return Livraison
     */
    public function setAdresse($adresse)
    {
        $this->adresse = $adresse;

        return $this;
    }

    /**
     * Get adresse
     *
     * @return string
     */
    public function getAdresse()
    {
        return $this->adresse;
    }

    /**
     * Set longitude
     *
     * @param float $longitude
     *
     * @return Livraison
     */
    public function setLongitude($longitude)
    {
        $this->longitude = $longitude;

        return $this;
    }

    /**
     * Get longitude
     *
     * @return float
     */
    public function getLongitude()
    {
        return $this->longitude;
    }

    /**
     * Set latitude
     *
     * @param float $latitude
     *
     * @return Livraison
     */
    public function setLatitude($latitude)
    {
        $this->latitude = $latitude;

        return $this;
    }

    /**
     * Get latitude
     *
     * @return float
     */
    public function getLatitude()
    {
        return $this->latitude;
    }

    /**
     * Set nomclient
     *
     * @param string $nomclient
     *
     * @return Livraison
     */
    public function setNomclient($nomclient)
    {
        $this->nomclient = $nomclient;

        return $this;
    }

    /**
     * Get nomclient
     *
     * @return string
     */
    public function getNomclient()
    {
        return $this->nomclient;
    }

    /**
     * Set nomlivreur
     *
     * @param string $nomlivreur
     *
     * @return Livraison
     */
    public function setNomlivreur($nomlivreur)
    {
        $this->nomlivreur = $nomlivreur;

        return $this;
    }

    /**
     * Get nomlivreur
     *
     * @return string
     */
    public function getNomlivreur()
    {
        return $this->nomlivreur;
    }

    /**
     * Get idLivraison
     *
     * @return integer
     */
    public function getIdLivraison()
    {
        return $this->idLivraison;
    }

    /**
     * Set idUser
     *
     * @param \AppBundle\Entity\Utilisateur $idUser
     *
     * @return Livraison
     */
    public function setIdUser(\AppBundle\Entity\Utilisateur $idUser = null)
    {
        $this->idUser = $idUser;

        return $this;
    }

    /**
     * Get idUser
     *
     * @return \AppBundle\Entity\Utilisateur
     */
    public function getIdUser()
    {
        return $this->idUser;
    }

    /**
     * Set idcommande
     *
     * @param \AppBundle\Entity\Commande $idcommande
     *
     * @return Livraison
     */
    public function setIdcommande(\AppBundle\Entity\Commande $idcommande = null)
    {
        $this->idcommande = $idcommande;

        return $this;
    }

    /**
     * Get idcommande
     *
     * @return \AppBundle\Entity\Commande
     */
    public function getIdcommande()
    {
        return $this->idcommande;
    }
}

File: mobileintegration/CodenameOneConsommationWebService/src/com/mycompany/Entite/Livraison.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.Entite;

import java.util.Date;

/**
 *
 * @author seifeddinebenfraj
 */
public class Livraison {
    private int ID_Livraison;
    public static int ID_Panier=1;
    private int ID_Pan;
    private float Montant;
    private int Etat;
    private Date Date;
    private String Adresse;
    private double Longitude;
    private double Latitude;
    public static int id_User=5;
    private int id_Us;
    private String nomLivreur;
    private String nomClient;

    public Livraison(float Montant, String Adresse, String nomClient) {
        this.Montant = Montant;
        this.Adresse = Adresse;
        this.nomClient = nomClient;
    }

    public Livraison() {
    }

    public Livraison(int ID_Livraison, int ID_Pan, float Montant, int Etat, Date Date, String Adresse, double Longitude, double Latitude, int id_Us, String nomLivreur) {
        this.ID_Livraison = ID_Livraison;
        this.ID_Pan = ID_Pan;
        this.Montant = Montant;
        this.Etat = Etat;
        this.Date = Date;
        this.Adresse = Adresse;
        this.Longitude = Longitude;
        this.Latitude = Latitude;
        this.id_Us = id_Us;
        this.nomLivreur = nomLivreur;
    }

    public Livraison(String Adresse, float Montant, int Etat) {
        this.Adresse = Adresse;
        this.Montant = Montant;
        this.Etat = Etat;
    }

    public Livraison(int ID_Livraison,String Adresse, float Montant, int Etat) {
        this.ID_Livraison = ID_Livraison;
        this.Montant = Montant;
        this.Etat = Etat;
        this.Adresse = Adresse;
    }

    public Livraison(String nomClient, String Adresse) {
        this.Adresse = Adresse;
        this.nomClient = nomClient;
    }
    
    


    public int getID_Livraison() {
        return ID_Livraison;
    }

    public void setID_Livraison(int ID_Livraison) {
        this.ID_Livraison = ID_Livraison;
    }

    public int getID_Panier() {
        return ID_Panier;
    }

    public void setID_Panier(int ID_Panier) {
        this.ID_Panier = ID_Panier;
    }

    public float getMontant() {
        return Montant;
    }

    public void setMontant(float Montant) {
        this.Montant = Montant;
    }

    public int getEtat() {
        return Etat;
    }

    public void setEtat(int Etat) {
        this.Etat = Etat;
    }

    public Date getDate() {
        return Date;
    }

    public void setDate(Date Date) {
        this.Date = Date;
    }

    public String getAdresse() {
        return Adresse;
    }

    public void setAdresse(String Adresse) {
        this.Adresse = Adresse;
    }

    public double getLongitude() {
        return Longitude;
    }

    public void setLongitude(double Longitude) {
        this.Longitude = Longitude;
    }

    public double getLatitude() {
        return Latitude;
    }

    public void setLatitude(double Latitude) {
        this.Latitude = Latitude;
    }

    public int getId_User() {
        return id_User;
    }

    public void setId_User(int id_User) {
        this.id_User = id_User;
    }

    public int getID_Pan() {
        return ID_Pan;
    }

    public void setID_Pan(int ID_Pan) {
        this.ID_Pan = ID_Pan;
    }

    public int getId_Us() {
        return id_Us;
    }

    public void setId_Us(int id_Us) {
        this.id_Us = id_Us;
    }

    public String getNomLivreur() {
        return nomLivreur;
    }

    public void setNomLivreur(String nomLivreur) {
        this.nomLivreur = nomLivreur;
    }

    public String getNomClient() {
        return nomClient;
    }

    public void setNomClient(String nomClient) {
        this.nomClient = nomClient;
    }

    @Override
    public String toString() {
        return "Livraison{" + "ID_Livraison=" + ID_Livraison + ", Montant="
                + Montant + ", Etat=" + Etat + ", Adresse=" + Adresse 
                + ", nomLivreur=" + nomLivreur + ", nomClient=" + nomClient + '}';
    }
    
    

    

    
    
    
    
    
}
